src/tinygp/kernels/quasisep.py

```diff
@@ -737,6 +741,177 @@ def transition_matrix(self, X1: JAXArray, X2: JAXArray) -> JAXArray:
         ).real
 
 
+@dataclass
+class carma(Quasisep):
```

Comment:
This should be `CARMA` to be consistent with the style, and once we get this sorted, it will replace the existing implementation.

Reply:
OK, do you suggest I make all my 'carma' upper case and remove (or comment out) the exiting implementation in the branch?

Reply:
Yep! You don't need to do that immediately, but it's what we'll want to do before merging.